extensions/positron-r/src/test/indentation.test.ts

```diff
@@ -40,13 +40,9 @@ suite('Indentation', () => {
 		// ** TODO **
 		//
 		// This test currently causes the entire test suite to be unexpectedly
-		// terminated. It's disabled on CI until we can figure out why.
-		//
-		// We use the `POSITRON_BUILD_NUMBER` environment variable to detect if
-		// we are running on CI; this is always set by the CI environment.
-		if (env['POSITRON_BUILD_NUMBER']) {
-			return;
-		}
```

Comment:
did you intend to remove the env var check?

Reply:
yeah, because it only prevented the test from running in CI.  when it fails in CI, the natural reaction is to try locally and its broken locally, too, so wanted to get it completely out